ui/apps/platform/src/Containers/Compliance/Dashboard/ManageStandardsModal.tsx

```diff
@@ -0,0 +1,121 @@
+import React, { ReactElement, useState } from 'react';
+import { Alert, Button, Checkbox, Form, Modal } from '@patternfly/react-core';
+import { useFormik } from 'formik';
+
+import {
+    ComplianceStandardMetadata,
+    fetchComplianceStandards,
+    // patchComplianceStandard,
+} from 'services/ComplianceService';
+import { getAxiosErrorMessage } from 'utils/responseErrorUtils';
+
+export type ManageStandardsModalProps = {
+    standards: ComplianceStandardMetadata[];
+    onCancel: () => void;
+    onSave: (standards: ComplianceStandardMetadata[]) => void;
+};
+
+/*
+ * Formik values have standard id as key and showScanResults as value.
+ * Therefore, negate hideScanResults value here and
+ * showComplianceScanMap[id] value in onSubmit function below.
+ */
+function getShowScanResultsMap(standards: ComplianceStandardMetadata[]): Record<string, boolean> {
+    return Object.fromEntries(standards.map(({ id, hideScanResults }) => [id, !hideScanResults]));
+}
+
+function ManageStandardsModal({ standards, onSave, onCancel }): ReactElement {
+    const [errorMessage, setErrorMessage] = useState('');
+    const { dirty, handleSubmit, isSubmitting, setFieldValue, setSubmitting, values } = useFormik<
+        Record<string, boolean>
+    >({
+        initialValues: getShowScanResultsMap(standards),
+        onSubmit: (/* showScanResultsMap */) => {
+            setErrorMessage('');
+            const patchRequestPromises = [];
+            /*
+            // Filter standards for which hideScanResults property has changed,
+            // and them map to promises for patch requests.
+            const patchRequestPromises = standards
+                .filter(
+                    ({ hideScanResults, id }) =>
+                        Boolean(hideScanResults) !== !showScanResultsMap[id]
+                )
+                .map(({ id }) => patchComplianceStandard(id, !showScanResultsMap[id]));
+            */
+
+            Promise.all(patchRequestPromises)
+                .then(() => {
+                    fetchComplianceStandards()
+                        .then((standardsFetchedAfterPatchRequests) => {
+                            onSave(standardsFetchedAfterPatchRequests);
+                        })
+                        .catch((error) => {
+                            setErrorMessage(getAxiosErrorMessage(error));
+                        })
+                        .finally(() => {
+                            setSubmitting(false);
+                        });
+                })
+                .catch((error) => {
+                    // TODO fetchComplianceStandards in case some succeed before one fails?
+                    setErrorMessage(getAxiosErrorMessage(error));
+                    setSubmitting(false);
+                });
+        },
+    });
+
+    return (
+        <Modal
+            title="Manage standards"
+            variant="medium"
```

Comment:
I think a Modal `variant` prop value of "small" looks better for this amount of data. (Also closer to the mock.)

https://www.patternfly.org/v4/components/modal#small-modal